### PR TITLE
update chart definition to include --notify-url

### PR DESCRIPTION
**Pull request by atighineanu**
Updating chart definition for --notify-url, following the PR: https://github.com/weaveworks/kured/pull/315

### CODE REVIEW

File: charts/kured/README.md
Comment (ckotzbauer) [inline charts/kured/README.md:59]: Small nit: Can you rename the parameter from "notifyURL" to "notifyUrl" to be consistent with e.g. "slackHookUrl"?

**Review by ckotzbauer — CHANGES_REQUESTED** [review]
Thanks for the PR @atighineanu!
This LGTM if the naming is addressed properly 😉

**Comment by ckotzbauer** [comment]
The merge of this PR has to wait until 1.7.0 is released.

File: charts/kured/README.md
Comment (atighineanu) [reply charts/kured/README.md:59]: Sure. I thought I'd keep the name consistent with the one in the main.go code.

File: charts/kured/Chart.yaml
Comment (yershalom) [inline charts/kured/Chart.yaml:2]: Should the `appVersion` changed too?

Comment (ckotzbauer) [reply charts/kured/Chart.yaml:2]: No not now. This will be done when we make the new kured release.

Comment (evrardjp) [reply charts/kured/Chart.yaml:2]: As you mentioned: This has to wait for the 1.7.0 release to merge, so maybe we can bump it to "1.7.0" already, though it will certainly fail until 1.7.0 is public. Which seems fair to me. :)

**Review by evrardjp — COMMENTED** [review]
Looks okay. I would already bump to 1.7.0, and wait for it to pass tests before merging.

But I have no strong opinion, I trust @ckotzbauer for this! :)

**Comment by ckotzbauer** [comment]
I'm also fine with what @evrardjp suggests to increase the `appVersion` to 1.7.0 already, as this wouldn't be merged until it is released. So maybe that's the more elegant solution 😉

File: charts/kured/Chart.yaml
Comment (atighineanu) [reply charts/kured/Chart.yaml]: Done.

**Comment by evrardjp** [comment]
I suppose the image also needs to be bumped to 1.7.0. It's currently located in Values.image.tag.

**Comment by evrardjp** [comment]
@ckotzbauer can we, by default, make sure that the image tag is set to the app version? That would make it simpler.

**Comment by ckotzbauer** [comment]
@evrardjp Yes, that's a good idea. I make a PR for that.

File: charts/kured/templates/daemonset.yaml
Comment (ckotzbauer) [inline charts/kured/templates/daemonset.yaml:92]: Please use `notifyUrl` here.

File: charts/kured/values.minikube.yaml
Comment (ckotzbauer) [inline charts/kured/values.minikube.yaml:18]: Please use `notifyUrl` here.

**Comment by evrardjp** [comment]
@ckotzbauer I wasn't sure if that was possible, but if it is, that's awesome! Maybe we can edit the Makefile to remove that sed in values.yaml then :)

File: charts/kured/values.yaml
Comment (ckotzbauer) [inline charts/kured/values.yaml:37]: Please use `notifyUrl` here.

File: charts/kured/Chart.yaml
Comment (evrardjp) [inline charts/kured/Chart.yaml:5]: This will be conflicting on rebase. Instead of bumping to 2.4.1, I would bump the version to 2.5.0, as we are moving to a new version of kured in practice.

**Review by evrardjp — CHANGES_REQUESTED** [review]
See comment above.

**Review by ckotzbauer — APPROVED** [review]

**Comment by evrardjp** [comment]
@atighineanu did you rebase before updating the value? It shows as still conflicting.

**Comment by dholbach** [comment]
Sorry. Can you please retarget your branch to be merged into `main` please?

**Comment by atighineanu** [comment]
> Sorry. Can you please retarget your branch to be merged into `main` please?

Sure.

**Pull request closed without merging**
